src/main/resources/application.conf

```diff
@@ -5,8 +5,7 @@ s3 {
 }
 
 signatures {
-  droid = ${DROID_SIGNATURE_VERSION}
-  container = ${CONTAINER_SIGNATURE_VERSION}
   cdn = "https://cdn.nationalarchives.gov.uk/documents"
+  version = "https://www.nationalarchives.gov.uk"
```

Comment:
Is `version` the best name for this parameter, bit confusing as its used as the national archives URL in the code.

Reply:
Good point, I think I originally had it as version in the code and it was confusing. I've changed it here.